component-tools/src/main/java/org/talend/sdk/component/tools/ComponentValidator.java

```diff
@@ -183,25 +178,20 @@ private String validateIcon(final Icon annotation, final Collection<String> erro
                     prefixes.add(s + File.separator + ICONS + "dark" + File.separator + icon);
                 });
                 svgs = prefixes.stream().map(s -> new File(s + ".svg")).collect(toSet());
-                pngs = prefixes.stream().map(s -> new File(s + "_icon32.png")).collect(toSet());
             }
 
-            svgs.stream()
```

Comment:
this part was checking icons for legacy folder
and on Cloud we still used legacy icons, not dark and light theme
@undx Do you have some remarks about it?

Reply:
"svg.streams()" is on line 187 now.